src/components/Header.tsx

```diff
@@ -86,13 +82,7 @@ export default function Header() {
           </a>
           {headerIcons.map((image, index) => (
             <span className="flex" key={index}>
-              <Image
-                src={image}
-                alt="mail"
-                width={10}
-                height={10}
-                style={{ height: "100%", width: "auto" }}
-              />
+              <Image src={image} alt="mail" width={16} height={16} />
```

Comment:
Does this fix the console warnings?

Reply:
fixed via providing aspect ratio to className.